tests/filecheck/dialects/comb/comb_ops.mlir

```diff
@@ -48,21 +48,21 @@
   %xor = comb.xor %lhsi32, %rhsi32 : i32
   // CHECK-NEXT: %xor = comb.xor %lhsi32, %rhsi32 : i32
 
+  %icmp = "comb.icmp"(%lhsi1, %rhsi1) {"predicate" = 2 : index, "two_state"} : (i1, i1) -> i1
+  // CHECK-NEXT: %icmp = comb.icmp slt, %lhsi1, %rhsi1 : i1
+
   %parity = "comb.parity"(%lhsi1) : (i1) -> i1
   // CHECK-NEXT: %parity = "comb.parity"(%lhsi1) : (i1) -> i1
 
-  %extract = "comb.extract"(%lhsi32) {"low_bit" = 1 : i32} : (i32) -> i32
-  // CHECK-NEXT: %extract = "comb.extract"(%lhsi32) {"low_bit" = 1 : i32} : (i32) -> i32
+  %extract = comb.extract %lhsi32 from 1 : (i32) -> i32
+  // CHECK-NEXT: %extract = comb.extract %lhsi32 : i32
 
   %concat = comb.concat %lhsi32, %rhsi32 : i32
   // CHECK-NEXT: %concat = comb.concat %lhsi32, %rhsi32 : i32
 
-  %replicate = "comb.replicate"(%lhsi32) : (i32) -> i32
-  // CHECK-NEXT: %replicate = "comb.replicate"(%lhsi32) : (i32) -> i32
-
   %mux = comb.mux %lhsi1, %lhsi32, %rhsi32 : i32
   // CHECK-NEXT: %mux = comb.mux %lhsi1, %lhsi32, %rhsi32 : i32
 
-  %icmp = "comb.icmp"(%lhsi1, %rhsi1) {"predicate" = 2 : index, "two_state"} : (i1, i1) -> i1
-  // CHECK-NEXT: %icmp = comb.icmp slt, %lhsi1, %rhsi1 : i1
+  %replicate = comb.replicate %lhsi32 : (i32) -> i32
+  // CHECK-NEXT: %replicate = comb.replicate %lhsi32 : i32
   }) : () -> ()
```

Comment:
```suggestion
  }) : () -> ()

```
Please add a newline at the end of the file. Could you please install precommit using the command in the Makefile to do this automatically?